app/assets/javascripts/lib/router.coffee

```diff
@@ -1,9 +1,24 @@
 class @Router extends Backbone.Router
   routes:
+    "demand/:sidebar(/:slide)" : "demand"
+    "costs/:sidebar(/:slide)"  : "costs"
+    "overview/:sidebar(/:slide)": "overview"
+    "supply/:sidebar(/:slide)" : "supply"
+    "flexibility/:sidebar(/:slide)" : "flexibility"
+    "data/:sidebar(/:slide)" : "data"
     "report" : "report"
     ":tab/:sidebar(/:slide)" : "load_slides"
     "" : "load_default_slides"
 
+  demand:  (sidebar, slide) => @load_slides('demand', sidebar, slide)
```

Comment:
@marliekeverweij , I don't really get the changes in these lines (13-21). Are you sure they need to be there?

Reply:
You're right! They should not be there. I've deleted them now.